src/Engine/GameAction.java

```diff
@@ -181,7 +182,7 @@ public CaptureAction(GameMap map, Unit actor, Path movePath)
         if( capture.willCapture() ) // If this will succeed, check if the CO will lose as a result.
         {
           // Check if capturing this property will cause someone's defeat.
-          if( (captureLocation.getEnvironment().terrainType.sustainsSide()) && (null != captureLocation.getOwner()) )
+          if( (captureLocation.getEnvironment().terrainType == TerrainType.HEADQUARTERS) && (null != captureLocation.getOwner()) )
```

Comment:
I still feel we should support the AWBW style of Lab use, at least somewhat.
Maybe that should just be the map converter turning Labs into HQs until we figure out better what we want, though?
(though having unit bans modified by Lab ownership is pretty cool too)

Reply:
Not sure how unit bans work, but my thoughts are: If we want to support alternate game modes (multiple HQs, capture all/just one to win, etc), we can do that. However, I think those rules are a property of the game mode, not of the TerrainType. We can also make new terrain types if we want, so there's no need to overload labs or whatever to support it.

Reply:
Unit bans can be either a ban or a "lab unit"
Lab units can only be built by players who own a lab currently.
So yes, it actually is kind of tied to the terrain type.
A decent number of AWBW maps mix players with only labs and players with only HQs so that only the lab players only have access to certain unit types, or maybe they'll have neutral labs that *can* be captured in order to gain access to the unit types in question.
Another point that may be worth mentioning is Labs don't give income, so a player can be given a number of them at the start (for e.g. a multi-capture required victory) without changing their day to day income.

Reply:
Labs are actually used as HQs in the carts (see: the lab missions), and the technology aspect is attractive also.
I'm mildly opposed to making Labs unable to take at least either role, due to map compatibility, but I guess it's not that big of a deal since find/replace is pretty easy.
That said, I'm not at all opposed to having other buildings with just one of the aspects.